scripts/convert_ipynb_to_mdx.py

```diff
@@ -0,0 +1,552 @@
+                        )
+                        components_output += (
+                            f'import { {component_name} } from "./{filename}.jsx";\n'
+                        ).replace("'", " ")
+                        mdx_output += f"<{component_name} />\n\n"
+                        # Parse the javascript for the bokeh JSON data.
+                        flag = "const docs_json = "
+                        json_string = list(
+                            filter(
+                                lambda line: line.startswith(flag),
+                                [
+                                    line
+                                    for line in [
+                                        line.strip()
+                                        for line in cell_output_data.splitlines()
+                                    ]
+                                ],
+                            )
+                        )[0]
+                        # Ignore the const definition and the ending ; from the line.
+                        json_string = json_string[len(flag) : -1]
+                        json_data = json.loads(json_string)
+                        # The js from bokeh in the notebook is nested in a single key,
+                        # hence the reason why we do this.
+                        json_data = json_data[list(json_data.keys())[0]]
+                        js = {}
+                        js["target_id"] = div_name
+                        js["root_id"] = json_data["roots"]["root_ids"][0]
+                        js["doc"] = {
+                            "defs": json_data["defs"],
+                            "roots": json_data["roots"],
+                            "title": json_data["title"],
+                            "version": json_data["version"],
+                        }
+                        js["version"] = json_data["version"]
+                        # Save the bokeh JSON data.
+                        file_path = str(plot_data_folder.joinpath(f"{div_name}.json"))
+                        with open(file_path, "w") as f:
+                            json.dump(js, f, indent=2)
+                        # Add a React component to the jsx output.
+                        jsx_output += (
+                            f"export const {component_name} = () => {{\n"
+                            "  return (\n"
+                            "    <BokehFigure pathToData={"
+                            f'"./assets/plot_data/{div_name}.json"}} />\n'
+                            "  );\n"
+                            "};\n\n"
+                        )
+
+            # Handle "execute_result".
+            if cell_output_type == "execute_result":
+                if data_category == "text":
+                    # Handle plain text.
+                    if data_type == "plain":
+                        mdx_output += f"```python\n>>> {cell_output_data}\n```\n\n"
```

Comment:
It's a bit hard to distinguish outputs from inputs when both of them are rendered with the same CSS style :).

One thing to consider is to leave the outputs as plain text. It'll be better if the outputs can have, say, a different color of background (similar to [how PyTorch tutorials are rendered](https://pytorch.org/tutorials/beginner/basics/quickstart_tutorial.html))], but don't worry too much about it if this takes too much of effort to do 😄